src/redux/books/books.js

```diff
@@ -24,18 +24,13 @@ export const addBook = (id, title, author, category) => () => {
   };
   fetch(baseUrl, post)
     .then((response) => console.log(response.status));
```

Comment:
- [ ] You have a couple of console statements. Kindly remove all of them so as to keep your code neat.
- [ ] (OPTIONAL) It would also be great if you could address the warnings on your vs code terminal and the browser terminal too 👍
- vs code terminal:
![Annotation 2022-06-17 054603](https://user-images.githubusercontent.com/85113567/174227454-96618b27-38c3-4b8a-8686-5530d885ff86.jpg)